tmt/tests/test-00-bootc-install.sh

```diff
@@ -36,6 +36,25 @@ if [ "$TMT_REBOOT_COUNT" -eq 0 ]; then
     ls -al "$BOOTC_TEMPDIR"
 
     pushd "$TMT_TREE"
+        if [ -e /etc/yum.repos./tag-repository.repo ]; then
```

Comment:
You are missing "d" in "yum.repos.d" directory name.

Reply:
Thank you! <facepalm>